src/uikit.cs

```diff
@@ -293,15 +293,15 @@ partial interface NSFileProviderExtension {
 	    [Static, Export ("writePlaceholderAtURL:withMetadata:error:")]
 	    bool WritePlaceholder (NSUrl placeholderUrl, NSDictionary metadata, ref NSError error);
 	
-		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'FileProvider.NSFileProviderManager.GetPlaceholderUrl (NSUrl)' instead.")]
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'FileProvider::NSFileProviderManager.GetPlaceholderUrl (NSUrl)' instead.")]
```

Comment:
heh, I still think `FileProvider::NSFileProviderManager.GetPlaceholderUrl` is right, let me explain my reasoning.

`NSFileProviderExtension` the class containing the deprecated methods is inside `UIKit` namespace and the suggested replacement `NSFileProviderManager` is inside `FileProvider` namespace so users seeing this advice are likely having existing code and might not be aware of the new `FileProvider` framework where the new replacement lives in so having `FileProvider::NSFileProviderManager.GetPlaceholderUrl` tells the user that the new object holding the replacement member lives inside `FileProvider` namespace. Does this make sense?